impl/src/main/java/org/ehcache/config/serializer/DefaultSerializationProviderConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.config.serializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.ehcache.internal.serialization.JavaSerializationProvider;
+import org.ehcache.spi.serialization.Serializer;
+import org.ehcache.spi.service.ServiceConfiguration;
+
+public class DefaultSerializationProviderConfiguration implements ServiceConfiguration<JavaSerializationProvider> {
+
+  private final Map<String, TypeSerializerConfig> serializerConfig = new HashMap<String, TypeSerializerConfig>();
+  
+  @Override
+  public Class<JavaSerializationProvider> getServiceType() {
```

Comment:
Actually this _may well_ be fine... I need to check. I wonder if we don't walk up the chain. I'll write a test for that stuff in order to document this properly once & for all!